robottelo/ssh.py

```diff
@@ -303,86 +110,43 @@ def command(
     :param int timeout: Time to wait for the ssh command to finish.
     :param connection_timeout: Time to wait for establishing the connection.
     """
-    # temporary workaround due to import order.
-    from robottelo.config import settings
-
-    hostname = hostname or settings.server.hostname
-    timeout = timeout or settings.server.ssh_client.command_timeout
-    connection_timeout = connection_timeout or settings.server.ssh_client.connection_timeout
-    with get_connection(
+    client = get_client(
         hostname=hostname,
         username=username,
         password=password,
-        key_filename=key_filename,
-        key_string=key_string,
-        timeout=connection_timeout,
+        timeout=timeout,
         port=port,
-    ) as connection:
-        return execute_command(cmd, connection, output_format, timeout, connection_timeout)
+    )
+    result = client.execute(cmd, timeout=timeout)
+    if output_format and result.status == 0:
+        if output_format == 'csv':
+            result.stdout = hammer.parse_csv(result.stdout) if result.stdout else {}
+        if output_format == 'json':
+            result.stdout = hammer.parse_json(result.stdout) if result.stdout else None
+    return result
 
 
-def execute_command(cmd, connection, output_format=None, timeout=None, connection_timeout=None):
-    """Execute a command via ssh in the given connection
+def is_ssh_pub_key(key):
```

Comment:
Note I moved this to `robottelo.helpers` in the initial commit to `unstable` branch:

https://github.com/SatelliteQE/robottelo/blob/unstable/robottelo/helpers.py#L209